tests/test_asset_connect.py

```diff
@@ -29,7 +29,7 @@ def test_storage_connection(self):
 
         conn = S3Connection()
         try:
-            bucket = conn.get_bucket(self.settings['aws_storage_bucket'])
+            bucket = conn.get_bucket(self.settings['veda_s3_hotstore_bucket'])
```

Comment:
@muzaffaryousaf The sandbox bucket is always different (to allow for parallel sandboxes), but I think that's a good impulse.